frontend/src/components/auth/Login.tsx

```diff
@@ -44,6 +45,13 @@ const Login = (): React.ReactElement => {
     return <Redirect to={HOME_PAGE} />;
   }
 
+  // For testing purposes
+  const filter = {
+    causes: [], // empty array will include all causes
+    frequencies: ["Monthly", "One-time"], // if not empty, only donations with these frequencies will be included
+    years: [],
+  };
+
```

Comment:
You might have also copied some extra commits from Jordan and Patrick's branch too, it seems. Can you look into removing these too?